src/test/regress/expected/ref_citus_local_fkeys.out

```diff
@@ -0,0 +1,202 @@
+-- foreign key from reference table to citus local table --
+---------------------------------------------------------------------
+-- create test tables
+CREATE TABLE citus_local_table(l1 int primary key);
+SELECT create_citus_local_table('citus_local_table');
+ create_citus_local_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE reference_table(r1 int);
+SELECT create_reference_table('reference_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO reference_table VALUES (3);
+NOTICE:  executing the command locally: INSERT INTO ref_citus_local_fkeys.reference_table_1506003 (r1) VALUES (3)
+-- show that we are checking for foreign key constraint while defining, this should fail
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1);
+NOTICE:  executing the command locally: SELECT worker_apply_inter_shard_ddl_command (1506003, 'ref_citus_local_fkeys', 1506002, 'ref_citus_local_fkeys', 'ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1);')
+ERROR:  insert or update on table "reference_table_1506003" violates foreign key constraint "fkey_ref_to_local_1506003"
+-- we do not support CASCADE / SET NULL / SET DEFAULT behavior in "ALTER TABLE ADD fkey reference_table (to citus_local_table)" commands
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1) ON DELETE CASCADE;
+ERROR:  cannot specify non-restrictive behaviors for foreign key constraints from reference tables to citus local tables
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1) ON DELETE SET NULL;
+ERROR:  cannot specify non-restrictive behaviors for foreign key constraints from reference tables to citus local tables
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1) ON DELETE SET DEFAULT;
+ERROR:  cannot specify non-restrictive behaviors for foreign key constraints from reference tables to citus local tables
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1) ON UPDATE CASCADE;
+ERROR:  cannot specify non-restrictive behaviors for foreign key constraints from reference tables to citus local tables
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1) ON UPDATE SET NULL;
+ERROR:  cannot specify non-restrictive behaviors for foreign key constraints from reference tables to citus local tables
+ALTER TABLE reference_table ADD CONSTRAINT fkey_ref_to_local FOREIGN KEY(r1) REFERENCES citus_local_table(l1) ON UPDATE SET DEFAULT;
+ERROR:  cannot specify non-restrictive behaviors for foreign key constraints from reference tables to citus local tables
```

Comment:
I think the term `non-restrictive` is not widely used. Can we use something simpler like:
```
foreign key constraints from reference tables to citus local tables can only be defined as NO ACTION or RESTRICT
```